libs/types/src/election.ts

```diff
@@ -351,30 +351,12 @@ export enum BallotPaperSize {
 export const BallotPaperSizeSchema: z.ZodSchema<BallotPaperSize> =
   z.nativeEnum(BallotPaperSize);
 
-/**
- * Specifies where the target mark appears in relation to the option text.
- */
-export enum BallotTargetMarkPosition {
-  Left = 'left',
-  Right = 'right',
-}
-
-/**
- * Schema for {@link BallotTargetMarkPosition}.
- */
-export const BallotTargetMarkPositionSchema: z.ZodSchema<BallotTargetMarkPosition> =
-  z.nativeEnum(BallotTargetMarkPosition);
-
 export interface BallotLayout {
   paperSize: BallotPaperSize;
-  layoutDensity?: number;
-  targetMarkPosition?: BallotTargetMarkPosition;
```

Comment:
Reasoning through the implications of this: Changing the bubble position or layout density will no longer change the election hash. Which means that, even after changing bubble position or layout density, ballots with the previous bubble position or layout density will still be scannable, assuming all else is unchanged. Is that correct? (Seems reasonable to me, just wanted to check my understanding)

Reply:
Changing these layout settings will change the actual ballot layout, which will change the gridLayouts in the election definition (which is what actually tells the scanner where the bubbles are), which will change the election hash.

Reply:
Ah perfect - hadn't considered the grid layouts tie-in. Makes sense 👍